sr_camera_pose_estimation/launch/calibration.launch

```diff
@@ -1,33 +1,45 @@
 <launch>
   <arg name="sim" default="false" />
   <arg name="depth" default="false" />
-	<arg name="marker_size" default="9.0" />
-	<arg name="max_new_marker_error" default="0.08" />
-	<arg name="max_track_error" default="0.0" />
-	<arg name="bundle_files" default="$(find sr_description_common)/ar_markers/ar_marker_0_1_2_3_a4.xml" />
+  <arg name="marker_size" default="9.0" />
+  <arg name="max_new_marker_error" default="0.08" />
+  <arg name="max_track_error" default="0.0" />
+  <arg name="bundle_files" default="$(find sr_description_common)/ar_markers/ar_marker_0_1_2_3_a4.xml" />
+  <arg name="continuous" default="false" />
+  <arg name="window_width" default="100" />
+  <arg name="camera_root_frame" default="camera_link" />
 
   <arg if="$(arg depth)" name="cam_image_topic" default="/camera/depth_registered/points" />
   <arg if="$(arg depth)" name="cam_info_topic" default="/camera/depth_registered/camera_info" />	
   <arg if="$(arg depth)" name="output_frame" default="camera_depth_optical_frame" />
   <arg if="$(arg depth)" name="alvar_node" default="findMarkerBundles" />
-  <arg if="$(arg depth)" name="med_filt_size" default=" 10" />
+  <arg if="$(arg depth)" name="med_filt_size" default="10" />
   <arg unless="$(arg depth)" name="cam_image_topic" default="/camera/rgb/image_raw" />
   <arg unless="$(arg depth)" name="cam_info_topic" default="/camera/rgb/camera_info" />	
   <arg unless="$(arg depth)" name="output_frame" default="camera_rgb_optical_frame" />
   <arg unless="$(arg depth)" name="alvar_node" default="findMarkerBundlesNoKinect" />
   <arg unless="$(arg depth)" name="med_filt_size" default="" />
   
-  <node name="ar_track_alvar" pkg="ar_track_alvar" type="$(arg alvar_node)" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)$(arg med_filt_size) $(arg bundle_files)" />
-
   <node pkg="tf" type="static_transform_publisher" name="ar_bundle_broadcaster" args="0.1 0.58 0 2.35619449019 0 0 ra_base_link ar_marker_bundle_static 1" />
   <node pkg="tf" type="static_transform_publisher" name="ar_tag_0_broadcaster" args="-0.05 0.055 0 0.0 0 0 ar_marker_bundle_static ar_marker_0_static 1" />
   
   <node pkg="sr_camera_pose_estimation" type="camera_transform_publisher.py" name="sr_camera_transform_publisher" output="screen" >
     <param name="ar_marker_topic" value="ar_pose_marker" />
     <param name="camera_image_topic" value="$(arg cam_image_topic)" />
-    <rosparam param="marker_static_tf_names"> ['ar_marker_0_static'] </rosparam>
-    <rosparam param="marker_ids"> [0] </rosparam>
-    <param name="camera_root_frame" value="camera_link" />
+    <param name="marker_static_tf_name" value="ar_marker_0_static" />
+    <param name="marker_id" value="0" />
+    <param name="camera_root_frame" value="$(arg camera_root_frame)" />
     <param name="desired_camera_parent_frame" value="ra_base_link" />
```

Comment:
Perhaps we should make this an argument as well. We might have another arm (or no arm).